packages/scripts/scripts/build.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+const { sync: spawn } = require( 'cross-spawn' );
+const { sync: resolveBin } = require( 'resolve-bin' );
+
+/**
+ * Internal dependencies
+ */
+const {
+	getCliArgs,
+	hasCliArg,
+	hasProjectFile,
+} = require( '../utils' );
+
+const args = getCliArgs();
+
+const hasWebpackConfig = hasCliArg( '--config' ) ||
+	hasProjectFile( 'webpack.config.js' ) ||
+	hasProjectFile( 'webpack.config.babel.js' );
+
+if ( hasWebpackConfig ) {
+	// Sets environment to production.
+	process.env.NODE_ENV = 'production';
+
+	const { status } = spawn(
+		resolveBin( 'webpack' ),
+		[ ...args ],
```

Comment:
is a new array really needed here?  Could you just do `getCliArgs()`?

Reply:
Yes, we can update 👍

Reply:
Done.